bin/run_experiment.py

```diff
@@ -51,11 +51,11 @@ def main(
 
 if __name__ == "__main__":
     # Define the directories containing the training and test data.
-    train_dir = "resources/data/openml/train"
-    test_dir = "resources/data/openml/test"
+    train_dir = "resources/data/tabzilla/train"
```

Comment:
we should pass it as arg to the script (`python file.py --input abc -- output xyz`)

Reply:
up to you tbh